modules/installation-azure-confidential-vms.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_azure/installing-azure-network-customizations
+
+:_content-type: PROCEDURE
+[id="installation-azure-confidential-vms_{context}"]
+= Enabling Confidential VMs
+
+:FeatureName: Confidential VM
+
+You can enable Confidential VMs for compute nodes, control plane nodes, or all nodes when installing your cluster. 
+
+include::snippets/technology-preview.adoc[]
+
+//commenting out the second encryption option until https://issues.redhat.com/browse/OCPBUGS-18379 is resolved
+////
+Confidential VMs can operate in two modes: 
+
+* Only encrypting the virtual machine guest state storage, which contains the security state of the virtual machine
+* Encrypting the virtual machine guest state storage and the operating system storage
+
+If you encrypt the operating system storage, you can use a platform-managed encryption key or a key you manage.
+////
+
+[IMPORTANT]
+====
+You can use confidential VMs with the following VM sizes:
```

Comment:
Maybe this first part doesn't really need to be in the admonition? Is it just the part about not being supported on 64-bit ARM architectures that is critical to be in the admonition?

If so, then you could pull out this list and put it into the regular text. And then after that have the important block. That would get around the back-to-back admonitions issue at least.